src/Pkg.jl

```diff
@@ -549,6 +550,7 @@ function __init__()
         end
     end
     OFFLINE_MODE[] = get(ENV, "JULIA_PKG_OFFLINE", nothing) == "true"
+    FANCY_PRINT[] = (stdout isa Base.TTY) && (get(ENV, "CI", nothing) != "true")
```

Comment:
Imo this should be a function on `io` and the `ctx.io` or e.g. `stderr` should be passed to it.

Reply:
Good point. Changed